### PR TITLE
[ios] Removed iOS 7 support.

### PR DESCRIPTION
**Pull request by igrechuhin**

### CODE REVIEW

**Comment by syershov** [comment]
LGTM

File: iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationLoginViewController.mm [hunk 1]
```diff
@@ -217,41 +217,30 @@ - (void)updateUI
...
-      UIPopoverPresentationController * popPresenter = [alertController popoverPresentationController];
-      popPresenter.barButtonItem = self.navigationItem.rightBarButtonItem;
-    }
-    [self presentViewController:alertController animated:YES completion:nil];
-  }
-}
-
-#pragma mark - UIActionSheetDelegate
-
-- (void)actionSheet:(UIActionSheet *)actionSheet clickedButtonAtIndex:(NSInteger)buttonIndex
```
Comment (bykoianko) [inline iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationLoginViewController.mm]: А эта ф-ция удалена не по ошибке?

**Comment by bykoianko** [comment]
DUDE JTTP

**Comment by bykoianko** [comment]
LGTM if it'll have been built.

**Pull request merged**